test/verify/check-machines

```diff
@@ -2678,6 +2678,36 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         checkAutostart("subVmTest1", True)
         checkAutostart("subVmTest2", False)
 
+    def testCPUModel(self):
+        b = self.browser
+
+        self.createVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual machines")
+        self.waitVmRow("subVmTest1")
+
+        self.goToVmPage("subVmTest1")
+
+        # Copy host CPU configuration
+        b.click("#vm-subVmTest1-cpu-model button")
+        b.select_from_dropdown("#cpu-model-select-group select", "host-passthrough")
+        b.click("#cpu-config-dialog-apply")
+        # Warning about about difference in persistent and non-persitent XML shoud appear
+        b.wait_present("#cpu-tooltip")
+        b.click("#vm-subVmTest1-action-kebab button")
+        b.click("#vm-subVmTest1-forceOff")
+        b.wait_in_text("#vm-subVmTest1-state", "Shut off")
+        # Warning should dissapear and changes should be visible in the UI
+        b.wait_not_present("#cpu-tooltip")
+        b.wait_in_text("#vm-subVmTest1-cpu-model .pf-c-description-list__text", "host")
+
+        # Choose manually a CPU model
+        b.click("#vm-subVmTest1-cpu-model button")
+        b.select_from_dropdown("#cpu-model-select-group select", "coreduo")
+        b.click("#cpu-config-dialog-apply")
+        b.wait_in_text("#vm-subVmTest1-cpu-model .pf-c-description-list__text", "custom (coreduo)")
```

Comment:
Could you try to also start the VM here? This would probably require to add some features to cpu element in domains xml:
```
    <feature policy='disable' name='ss'/>
    <feature policy='disable' name='x2apic'/>
    <feature policy='disable' name='hypervisor'/>
    <feature policy='disable' name='monitor'/>
```

Reply:
It would be great if you could verify that it was set up outside of cockpit - with virt-inspect ot whatever cli tool can show that.

Reply:
@skobyda virt-manager has the same bahavior like this PR, at least from my understanding. 
Do you mind if this is true, improvements like this to come up in a followup?

Reply:
virt-xml pupolates there correctly